targets/nss/build.sh

```diff
@@ -59,3 +59,10 @@ for fuzzer in $FUZZERS; do
      /work/nss/lib/libnspr4.a /work/nss/lib/libplc4.a /work/nss/lib/libplds4.a \
      /work/nss/lib/prlog2.o -o /out/$fuzzer $FUZZER_LDFLAGS
 done
+
+# Archive and copy to /out seed corpus if the build succeeded.
+zip /work/nss/all_nss_seed_corpus.zip /src/nss-corpus/{cert,pkcs8,spki}/*
```

Comment:
I had some concerns in case if structure of the corpus repository changes (adding sub-directories, for example), but looks like libFuzzer reads corpus recursively. I'll use `/*/*`, thanks!